### ISSUE TITLE
When using aria-activedescendant, the focused element is not scrolled into the visible area

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria-practices#2052]
[Managing Focus in Composites Using aria-activedescendant](https://w3c.github.io/aria-practices/#kbd_focus_activedescendant) say:

> Move the visual focus indicator and, if necessary, scrolled the active element into view

Unfortunately, this is not even considered in the APG examples like https://w3c.github.io/aria-practices/examples/radio/radio-activedescendant.html, https://w3c.github.io/aria-practices/examples/menu-button/menu-button-actions-active-descendant.html and https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html).

When do I notice this: e.g. when I zoom out to 320x265 px according to WCAG 1.4.10 and then navigate through the entries with the arrow keys. In some APG examples, the problem would occur even more often and even without zoom if there were more elements (e.g. radio buttons).

Related: https://github.com/w3c/aria-practices/issues/1135, https://github.com/w3c/wcag/issues/2049#issuecomment-933437898

**Comment by alastc** [comment]
This seems like something that should be raised in the aria-practices issues? Or possibly as browser bugs.

**Comment by JAWS-test** [comment]
> This seems like something that should be raised in the aria-practices issues? 

That's what it is

> Or possibly as browser bugs.

I don't think so. If browsers are expected to focus aria-activedescendant correctly automatically, then there wouldn't be the sentence I quoted in ARIA APG that web designers have to do this

**Comment by alastc** [comment]
Apologies, I get so many WCAG-issues emails, I didn't notice the change in repo. No wonder I couldn't tag it!

**Comment by mcking65** [comment]
@JAWS-test 

This is an excellent observation.

@jesdaigle @smhigley Would it be possible to automate testing for visibility of elements given AT focus via aria-active descendant after changing view port size as part of our regression testing?

Given we have done code quality work on several of these examples recently, we should discuss fixes for this. Adding to agenda.

**Comment by a11ydoer** [comment]
@jongund  will look into this issue -  APG meeting on Oct 5, 2021.

**Comment by a11ydoer** [comment]
@jesdaigle will look into changing view port size as part of the regression testing. - APG meeting on Oct 5, 2021.
